doc/todo/NEWS.md

```diff
@@ -48,52 +48,63 @@ You can also read the news [on our website](https://www.libelektra.org/news/0.8.
 ### <<HIGHLIGHT2>>
 
 
-## Other New Features
-
-We added even more functionality, which could not make it to the highlights:
+## Bindings
 
 - <<TODO>>
 - <<TODO>>
 - <<TODO>>
 
 
-## New Plugins
+## Plugins
 
 - <<TODO>>
 - <<TODO>>
 - <<TODO>>
 
 
-## Other News
+## Tools
 
 - <<TODO>>
 - <<TODO>>
 - <<TODO>>
 
 
-## Documentation
+## Scripts
+
+- <<TODO>>
+- <<TODO>>
+- <<TODO>>
+
 
-We improved the documentation in the following ways:
+## Support
```

Comment:
What should be written here? (Except of shell completion features, which would also fit to scripts.)

Reply:
> What should be written here? 

Stuff that does not fit in the other categories? This update was mainly a reaction to your statement from above:

> Maybe you can avoid "Other News" even more?

.